test/crit/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/checkpoint-restore/go-criu/v5/crit"
+)
+
+func main() {
+	imgs := getImgs()
+	for _, img := range imgs {
+		log.Println("=== ", img)
+		testImg := fmt.Sprintf("%s.test.img", filepath.Base(img))
+		c := crit.New(img, testImg, "", false, false)
+		// Decode the binary image file
+		decodedImg, err := c.Decode()
+		if err != nil {
+			log.Fatal("[DECODE]: ", err)
+			continue
```

Comment:
`log.Fatal()` would call `os.Exit(1)`, and thus `continue` is redundant?

Reply:
Oops! I had added `continue` with `log.Println()` when I first wrote it, will remove 👍